client/src/main/java/com/alibaba/nacos/client/naming/NacosNamingMaintainService.java

```diff
@@ -63,12 +76,26 @@ private void init(Properties properties) throws NacosException {
         namespace = InitUtils.initNamespaceForNaming(properties);
         InitUtils.initSerialization();
         InitUtils.initWebRootContext(properties);
-        ServerListManager serverListManager = new ServerListManager(properties, namespace);
-        SecurityProxy securityProxy = new SecurityProxy(properties,
+        serverListManager = new ServerListManager(properties, namespace);
+        securityProxy = new SecurityProxy(properties,
                 NamingHttpClientManager.getInstance().getNacosRestTemplate());
+        initSecurityProxy();
         serverProxy = new NamingHttpClientProxy(namespace, securityProxy, serverListManager, properties, null);
     }
     
+    private void initSecurityProxy() {
+        this.executorService = new ScheduledThreadPoolExecutor(1, r -> {
+            Thread t = new Thread(r);
+            t.setName("com.alibaba.nacos.client.naming.maintainService.security");
+            t.setDaemon(true);
+            return t;
+        });
+        this.securityProxy.login(serverListManager.getServerList());
+        this.executorService.scheduleWithFixedDelay(() -> securityProxy.login(serverListManager.getServerList()), 0,
```

Comment:
Is it always login?

Reply:
Yes, in order to refresh token. Refer to NamingClientProxyDelegate.java， you'll see same code.